core/js/typespecs/api_types.ts

```diff
@@ -170,6 +170,13 @@ function generateBaseEventOpSchema(objectType: ObjectTypeWithEvent) {
       .describe(
         "Human-identifying attributes of the span, such as name, type, etc.",
       ),
+    origin: z
+      .object({
+        object_type: z.string(),
+        object_id: z.string().uuid(),
```

Comment:
should we include descriptions?

Reply:
feel free to steal these: https://github.com/braintrustdata/braintrust-sdk/blob/custom-viewers/core/js/typespecs/app_types.ts#L342

Reply:
Done